exonum/src/events/noise/tests.rs

```diff
@@ -0,0 +1,77 @@
+#[cfg(test)]
+mod test {
+    use snow::NoiseBuilder;
+    use snow::types::Dh;
+    use snow::wrappers::crypto_wrapper::Dh25519;
+
+    use crypto::PUBLIC_KEY_LENGTH;
+    use crypto::{convert_keypair_for_handshake, gen_keypair};
+
+    #[test]
+    fn test_convert_ed_to_curve_dh() {
```

Comment:
Don't you think that this test will look better in `crypto.rs`? It's not about noise proto, it's about conversion logic.